src/main/java/seedu/address/logic/commands/DeleteCommand.java

```diff
@@ -27,8 +27,10 @@ public class DeleteCommand extends Command {
             + "Example: " + COMMAND_WORD + " 1 or " + COMMAND_WORD + " " + PREFIX_NAME + "John Doe";
 
 
-    public static final String MESSAGE_DELETE_HELP = "Delete Command: (Coming soon)\n"
-            + "Format: (Coming soon)\n";
+    public static final String MESSAGE_DELETE_HELP = "Delete Command: Deletes a client in the Realodex by "
+            + "either the client's full name or index number.\n"
```

Comment:
Hmm, maybe we could specify the index in the current list

Reply:
Yes, 
```suggestion
            + "either the client's full name or index number shown in the list.\n"
```

Reply:
index number only!